toolkit/offchain/src/update_governance/test.rs

```diff
@@ -131,7 +131,7 @@ fn test_update_governance_tx() -> Transaction {
 		test_values::VERSION_ORACLE_VALIDATOR,
 		test_values::VERSION_ORACLE_POLICY,
 		genesis_utxo().to_domain(),
-		new_governance_authority(),
+		vec![new_governance_authority()],
```

Comment:
you should make some decision regarding ownership, either you provide governance by owned vec and always utilize it or provide it as a reference or slice and utilize it causally. For that usecase, I propose to pass it by reference/slice